components/Dialogs/src/MDCAlertController.m

```diff
@@ -169,6 +169,54 @@ - (void)addAction:(MDCAlertAction *)action {
   }
 }
 
+- (void)setTitleFont:(UIFont *)titleFont {
+  self.alertView.titleFont = titleFont;
```

Comment:
This won't work if the self.alertView hasn't been instantiated yet.
If self.alertView don't exist yet, we will need to store these values locally and apply them when alertView is instatiated.

Reply:
We are forcing the view to load when asked for alertView, I also have tests that make sure this behavior works.